cmd/root.go

```diff
@@ -72,4 +72,15 @@ func init() {
 			DefaultUsername: "apm_system",
 		},
 	})
+	// remove dashboard from export commands
+	for _, cmd := range RootCmd.ExportCmd.Commands() {
+		if cmd.Name() == "dashboard" {
+			RootCmd.ExportCmd.RemoveCommand(cmd)
+		}
+	}
+	// remove dashboards and ml-jobs from setup commands
+	setup := RootCmd.SetupCmd
+	setup.ResetFlags()
```

Comment:
Running `./apm-server setup` prints
```Loaded index template
Loading dashboards (Kibana must be running and reachable)
Skipping loading dashboards, No directory /home/juan/go/src/github.com/elastic/apm-server/kibana/6
```
which might confuse users (looks a bit like an error, when it is not)

Also `./apm-server help` references dashboards.

Reply:
I will investigate, I am sure this worked when I tried it out.

Reply:
I checked again, you are right that it still shows up in the description, it was only removed in the flags listed in the help. I'll push a follow up.

Reply:
depends on https://github.com/elastic/beats/pull/10277/